Documentation/fleet-scaling.md

```diff
@@ -55,3 +55,9 @@ wins:
 * Making some defaults exported and allow them to be overridden. For instance
     fleet's tokenLimit controls how many Units are listed per "page". *See the
     `--token-limit` flag.*
+
```

Comment:
Er, sorry. I thought I commented here but apparently not. Can you remove lines 36-46 from this file?